src/test/java/com/loofah/graph/api/integrationTests/DatabaseSeeder.java

```diff
@@ -20,30 +20,39 @@
 
     public static final List<Skill> SKILLS = Collections.unmodifiableList(Arrays.asList(
```

Comment:
I've put these in reverse order to allow testing of the sorting behaviour - open to suggestions for neater ways to do that

Reply:
i can't think of another way but i think this is ok